tests/integration/requiredStandards.intTest.ts

```diff
@@ -59,6 +59,28 @@ describe("Defects Service", () => {
             expect(res.body).toBe("euVehicleCategory required");
           });
       });
+
+      it("a validation error should be produced where there multiple duplicated query parameters", async () => {
+          const expectedResponse = JSON.parse(
```

Comment:
The expectedResponse isn't being used in this test so I don't think it's required.